### PR TITLE
Add podspec to DataSourceKit

### PR DESCRIPTION
**Pull request by dehlen**
This way the framework is also usable via CocoaPods.

### CODE REVIEW

File: DataSourceKit.podspec [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+  # ――― Source Code ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+  #
+  #  CocoaPods is smart about how it includes source code. For source files
+  #  giving a folder will include any swift, h, m, mm, c & cpp files.
+  #  For header files it will include any header in the folder.
+  #  Not including the public_header_files will make all headers public.
+  #
+
+  s.source_files  = "DataSourceKit", "DataSourceKit/**/*.{h,m}"
```
Comment (evermeer) [inline DataSourceKit.podspec]: should also add swift files

Comment (dehlen) [reply DataSourceKit.podspec]: Sure I forgot to edit this.

**Comment by ishkawa** [comment]
👍 

Thank you for the help!

**Comment by ishkawa** [comment]
Just published on the CocoaPods website 🎉 
https://cocoapods.org/pods/DataSourceKit

**Pull request merged**